WcaOnRails/app/views/layouts/_navigation.html.erb

```diff
@@ -141,6 +141,13 @@
               </li>
               <li><%= link_to t('.edit_profile'), profile_edit_path %></li>
 
+              <li class="divider"></li>
+              <li><%= link_to t('.my_competitions'), my_comps_path %></li>
+
+              <% if current_user.wca_id? %>
```

Comment:
You could rewrite this line and the two below as just `<li><%= link_to t('.my_results'), person_path(current_user.wca_id) if current_user.wca_id? %></li>`, although I don't know if that would make it too long, so I'm fine leaving it as is!

Reply:
Ah, yep forgot Ruby can do that! Since functionally it's the same, tempted to leave it as-is as well. 
Since I'm not on WST, I can't merge/deploy. Would you be able to for me?